connectors/src/lib/dpdf2text.ts

```diff
@@ -0,0 +1,39 @@
+import { spawn } from "child_process";
+
+export async function dpdf2text(pdf_path: string): Promise<string> {
+  const args: string[] = [];
```

Comment:
can we / should we initialize the array with the stuff below instead of `args.push` ?